scripts/postinstall.js

```diff
@@ -84,6 +84,15 @@ const run = async () => {
       },
     ])
   );
+  //ToDo: Remove when opensearch-js is released to include https://github.com/opensearch-project/opensearch-js/pull/889
+  promises.push(
+    patchFile('node_modules/@opensearch-project/opensearch/lib/Serializer.js', [
+      {
+        from: 'val < Number.MAX_SAFE_INTEGER',
+        to: 'val < Number.MIN_SAFE_INTEGER',
+      },
+    ])
+  );
```

Comment:
2.13.0 been released https://github.com/opensearch-project/opensearch-js/releases/tag/2.13.0, guess we can remove this?